src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -87,9 +94,25 @@ protected override SyntaxNode GetObjectCreationNewExpression(SyntaxTree tree, in
             return base.GetDisplayAndInsertionText(symbol, context);
         }
 
-        protected override CompletionItemRules GetCompletionItemRules()
+        private static readonly CompletionItemRules s_objectRules = 
+            CompletionItemRules.Create(commitCharacterRules: ImmutableArray.Create(CharacterSetModificationRule.Create(CharacterSetModificationKind.Replace, " ([")));
```

Comment:
nit, consider wrapping these long lines.  If a line can't be seen in github (in *wide* mode no less), it's too long. :)